template-compose/buildSrc/src/main/java/Versions.kt

```diff
@@ -21,12 +21,11 @@ object Versions {
 
     const val GRADLE = "8.1.2"
 
-    const val HILT = "2.48"
-    const val HILT_NAVIGATION_COMPOSE = "1.0.0"
-
-    const val JAVAX_INJECT = "1"
     const val JUNIT = "4.13.2"
 
+    const val KOIN = "3.3.2"
+    const val KOIN_ANDROID = "3.3.2"
+    const val KOIN_ANDROIDX_COMPOSE = "3.4.1"
```

Comment:
There is a newer version `3.5.0` available for these three dependencies 🙇🏻

Reply:
Fixed in https://github.com/nimblehq/android-templates/pull/510/commits/b5f1fd6b92bed5b92b555c5373ba45667acd50da